kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -163,6 +175,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, src *v1alpha1.KafkaSourc
 		logging.FromContext(ctx).Error("Unable to create the receive adapter", zap.Error(err))
 		return err
 	}
+	r.scaleKafkaSource(ctx, ra, src)
```

Comment:
If a user is going to potentially mark the minimum replica count as 0, it would probably be good to add a status line (or at the very least, we should log) whether or not the `scaleKafkaSource` returned an error or succeeded.  It'd be a poor user experience to simply fail (and have 0 replicas)

Reply:
changed to return create ScaledObejct and log error